helpers/deploymentConfig.ts

```diff
@@ -4678,6 +4690,160 @@ export const globalConfig: NetworkConfig = {
     ],
     preconfiguredAddresses: preconfiguredAddresses.zksyncsepolia,
   },
+  opsepolia: {
+    tokensConfig: [
+      {
+        isMock: true,
+        name: "USD Coin",
+        symbol: "USDC",
+        decimals: 6,
+        tokenAddress: ethers.constants.AddressZero,
+      },
+      {
+        isMock: true,
+        name: "Tether USD",
+        symbol: "USDT",
+        decimals: 6,
+        tokenAddress: ethers.constants.AddressZero,
+      },
+      {
+        isMock: false,
+        name: "Wrapped Ether",
+        symbol: "WETH",
+        decimals: 18,
+        tokenAddress: "0x4200000000000000000000000000000000000006",
+      },
+      {
+        isMock: true,
+        name: "Wrapped BTC",
+        symbol: "WBTC",
+        decimals: 8,
+        tokenAddress: ethers.constants.AddressZero,
+      },
+      {
+        isMock: true,
+        name: "Optimism",
+        symbol: "OP",
+        decimals: 18,
+        tokenAddress: ethers.constants.AddressZero,
+      },
+      {
+        isMock: true,
+        name: "Venus",
+        symbol: "XVS",
+        decimals: 18,
+        tokenAddress: ethers.constants.AddressZero,
+      },
```

Comment:
The `XVS` token is not a mock token. It has to be deployed in the [token-bridge](https://github.com/VenusProtocol/token-bridge) project, and configured to be "bridgeable".

Reply:
```suggestion
      {
        isMock: false,
        name: "Venus",
        symbol: "XVS",
        decimals: 18,
        tokenAddress: "0x789482e37218f9b26d8D9115E356462fA9A37116",
      },
```

Reply:
Removed Mock token as well as it's deployment files from here, bc607576f1af61fb2aa1fca9d96a1ac67b0cb977